parsec/core/ipcinterface.py

```diff
@@ -18,10 +20,13 @@
     SerdeError,
     MsgpackSerializer,
 )
+from parsec.utils import open_service_nursery
 
 
 logger = get_logger()
 
+CommandHandlerCallback = Callable[[Dict[str, object]], Awaitable[Dict[str, Any]]]
```

Comment:
```suggestion
CommandHandlerCallback = Callable[[dict[str, object]], Awaitable[dict[str, Any]]]
```

using `dict` instead of `Dict` allow use to not import one element from typing (same for `list` & `tuple`)

Reply:
Done.